tools/BuildGraph/DGMLFormatter.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace BuildGraph
+{
+    public class DGMLFormatter : GraphFormatter
```

Comment:
Any C# code we need to review in detail? Or was the build train code already reviewed thoroughly?